packages/vega-typings/types/spec/axis.d.ts

```diff
@@ -182,7 +182,7 @@ export interface BaseAxis {
   /**
    * Horizontal text alignment of axis titles.
    */
-  titleAlign?: AlignValue;
+  titleAlign?: AlignValue | SignalRef;
```

Comment:
Isn't alignValue already a signal?

Reply:
AlignValue > AlignValueRef > ScaledValueRef > BaseValueRef > SignalRef

Reply:
ok right, then we don't need to do this?

Reply:
Yep